features/dd-sdk-android-flags-openfeature/src/test/kotlin/com/datadog/android/flags/openfeature/DatadogFlagsProviderFactoryTest.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.flags.openfeature
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+/**
+ * Integration tests for DatadogFlagsProvider factory methods.
+ *
+ * Note: These tests validate the factory method APIs exist and are callable.
+ * Full integration testing requires a configured Datadog SDK instance and is
+ * performed in instrumented tests.
+ */
+internal class DatadogFlagsProviderFactoryTest {
```

Comment:
I don't think all these tests are useful, given that we have API surface files for Kotlin and Java layout.

They just add maintenance burden giving nothing on top of what we see in API surface definition.